.github/workflows/windows.yml

```diff
@@ -36,16 +36,49 @@ jobs:
   unit_tests:
     strategy:
       matrix:
-        go-versions: [1.13.x]
+        go: ['1.16']
         platform: [windows-latest]
     runs-on: ${{ matrix.platform }}
     steps:
       - name: Install Go
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v2
         with:
-          go-version: ${{ matrix.go-version }}
+          go-version: ${{ matrix.go }}
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Run Windows Unit Tests
         run: |
           go test -v -race ./pkg/...
+
+  bump_version_test:
+    strategy:
+      matrix:
+        go: ['1.16']
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Install Tools
+        run: |
+          echo "${HOME}/.local/bin" >> $GITHUB_PATH
+          echo "/home/runner/work/csi-proxy/csi-proxy/go/bin" >> $GITHUB_PATH
+          PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+          curl -LO $PB_REL/download/v3.15.8/protoc-3.15.8-linux-x86_64.zip
+          unzip protoc-3.15.8-linux-x86_64.zip -d $HOME/.local
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          path: go/src/github.com/kubernetes-csi/csi-proxy
+      - name: Run bump version script
+        env:
+          GOPATH: /home/runner/work/csi-proxy/csi-proxy/go
+          GOBIN: /home/runner/work/csi-proxy/csi-proxy/go/bin
+        run: |
+          go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
+          protoc --version
+          cd /home/runner/work/csi-proxy/csi-proxy/go/src/github.com/kubernetes-csi/csi-proxy
+          API_GROUP=filesystem OLD_API_VERSION=v1beta2 NEW_API_VERSION=v99 scripts/bump-version.sh
```

Comment:
not very familiar this part, just want to make sure the version number is the right one,

Reply:
yes, this is actually new, so to bump the APIs I created the script `scripts/bump-version.sh` that automated the process which I used for all of the API versions that we needed to bump. I also added a CI script for it, the nice part is that this is also making sure that we don't break the generator code in the future too :)

The version number v99 is the arbitrary but it's ok for CI, it can't be v1 because we already have v1